packages/manager/modules/dbaas-logs/src/logs/detail/streams/add/logs-streams-add.html

```diff
@@ -131,7 +131,7 @@
             <oui-checkbox
                 id="stream_enable_compression"
                 name="stream_enable_compression"
-                data-description="{{:: 'logs_streams_enable_archieve_description' | translate}}"
+                data-description="{{ ctrl.getColdStoragePrice() }}"
```

Comment:
As any values change here
```suggestion
                data-description="{{:: ctrl.getColdStoragePrice() }}"
```

Reply:
Hello, thank you for your suggestion, however if I'm applying it, price isn't displayed.